xml/core_registers.xml

```diff
@@ -15,7 +15,17 @@
           15: There is external debug support, but it does not conform to any
           available version of this spec.
         </field>
-        <field name="0" bits="27:16" access="R" reset="0" />
+        <field name="0" bits="27:18" access="R" reset="0" />
+        <field name="progbufjump" bits="17:16" access="R" reset="Preset">
+            0: All jumps/branches are supported in the Program Buffer.
```

Comment:
Is there a RISC-V way of referring to this? To be really concrete about what a jump or branch instruction is?

Reply:
Looks like the ISA spec calls them Control Transfer Instructions.